lib/drivers/neutrino-dollar.js

```diff
@@ -0,0 +1,61 @@
+const Driver = require('../models/driver');
+const Supply = require('../models/supply');
+
+/**
+ * NeutrinoDollar driver. Supports circulating  and
+ * total supply for USDN token on Waves blockchain.
+ *
+ * @memberof Driver
+ * @augments Driver
+ */
+class NeutrinoDollar extends Driver {
+  constructor(options) {
+    super({
+      timeout: 100, // 10 request per second
+      supports: {
+        circulating: true,
+        total: true,
+        blockchains: ['WAVES'],
```

Comment:
The blockchain is not needed here. It doesn't have support for all the tokens on WAVES, just Neutrino 🙂 
```suggestion
```